docs/content/community/bijeenkomsten/20181025-demo-sprint-5.md

```diff
@@ -12,8 +12,39 @@ Amsterdam, 25 oktober 2018
 Slides:
 
 * [ZDS2 Demo Spint 5.pdf](/community/bestanden/zds2-demo-sprint-5.pdf)
-* [Opsplitsen RGBZ](/community/bestanden/opsplitsen-rgbz.pdf))
+* [Opsplitsen RGBZ](/community/bestanden/opsplitsen-rgbz.pdf)
 
 ## Feedback
 
-[aanvullen]
+### Wat is de rol en scope van mapper service? Moet dit gestandaardiseerd worden? ###
+todo
+
+### Hoe staat het met authenticatie en autorisatie? ###
+todo
+
+### Hoe gaan jullie om met externe gebruikers m.b.t. authenticatie en autorisatie? ###
+todo
+
+### De URL werkt niet bij update!? ###
+todo
+
+### Wat is er concreet op 20 december en wat is het vervolg? Stip op de horizon? ###
+todo
+
+### Wordt Docker veel gebruikt binnen gemeenten? ###
+todo
+
+### Wat is de huidige toepasbaarheid van de API's? ###
+todo
+
+### Wat was de concrete doelstelling(en) aan het begin van het traject? ###
+Om Zaakgericht Werken een stap verder te brengen worden Zaak- en Documentservices (ZDS) versie 2 ontwikkeld. Hierbij wordt een andere vorm van standaardisatie toegepast. Op basis van relevante informatiemodellen (RGBZ 2.0 en ImZTC 2.2) wordt met zowel publieke als private partijen in een agile proces vorm gegeven aan RESTful API’s die concreet invulling geven aan de gewenste standaard. De standaard wordt tegelijk met een referentie-implementatie ontwikkeld om de implementeerbaarheid aan te tonen, en als referentie te dienen voor latere implementaties.
+
+### Kunnen jullie agenda meesturen in de uitnodiging? ###
+Op de website waar de bijeenkomsten worden aangekondigd en in de definitieve bevestiging wordt een link naar de activiteiten van de huidige sprint opgenomen. https://github.com/VNG-Realisatie/gemma-zaken/projects/3
+
+### Wat verstaan jullie onder backward compatible? ###
+todo
+
+### Hoe ziet de implementatie eruit? ###
+todo
```

Comment:
Begrijp de vraag niet.

Reply:
Ik was er zelf helaas niet bij vanwege herfstvakantie maar misschien wil de vraagsteller weten waar de source code van de diverse componenten te vinden is. 

* https://github.com/vng-Realisatie/gemma-zaakregistratiecomponent voor de zrc.
* https://github.com/vng-Realisatie/gemma-documentregistratiecomponent voor drc.
* Etc.

Reply:
Vraag is voor mij ook onduidelijk